contrib/pylightning/lightning/lightning.py

```diff
@@ -507,9 +507,9 @@ def feerates(self, style, urgent=None, normal=None, slow=None):
         }
         return self.call("feerates", payload)
 
-    def fundchannel(self, node_id, satoshi, feerate=None, announce=True, minconf=None, utxos=None):
+    def fundchannel(self, node_id, amount, feerate=None, announce=True, minconf=None, utxos=None):
```

Comment:
I am uncertain about python, but would this not break existing Python code that uses this library but explicitly uses `satoshi=` syntax to call `fundchannel` as well?

Reply:
Thank you. I find an example,  `close`:
https://github.com/ElementsProject/lightning/blob/9578fb4035d14690bd2ae8d9788cbb05486dbeaa/contrib/pylightning/lightning/lightning.py#L316-L354

Reply:
Rename it directly is reckless.